packages/legacy/core/App/components/inputs/PINInput.tsx

```diff
@@ -51,6 +51,7 @@ const PINInputComponent = (
       justifyContent: 'flex-start',
       alignItems: 'center',
       ...PINInputTheme.cell,
+      borderColor: PINInputTheme.cell.backgroundColor
```

Comment:
Perhaps this is a good opportunity to add a `PINInputTheme.codeFieldRoot` field since Bifold users have a need to style it for theming and it is a separate element from the cell.

Reply:
@bryce-mcmath @MosCD3 I will close my PR as it conflicts with Mostafa's PR. However, this change in PINInput particularly, I will make changes to in separate PR @bryce-mcmath